src/ngraph/runtime/reference/quantize.hpp

```diff
@@ -44,11 +47,64 @@ namespace ngraph
                 {
                     Coordinate scale_offset_coord = project(input_coord, axes);
 
-                    // apply scale and offset
-                    REAL qvalue =
-                        std::round(input[input_transform.index(input_coord)] /
-                                   scale[scale_offset_transform.index(scale_offset_coord)]) +
-                        offset[scale_offset_transform.index(scale_offset_coord)];
+                    // apply scale
+                    REAL qvalue = input[input_transform.index(input_coord)] /
+                                  scale[scale_offset_transform.index(scale_offset_coord)];
+
+                    auto fe_round_mode = std::fegetround();
```

Comment:
I'd feel more comfortable if you guarded this with RAII. If this setting manages to escape the function, it's really really hard to figure out why the math has become subtly different,